### PR TITLE
removed the http check in navigateTosource

### PR DESCRIPTION
**Pull request by TWith2Sugars**
Since it doesn't seem to do anything other than a check I removed the use of the http client.

Everything works the same still just the browser pops up a little faster.

If we are to keep this then perhaps we need to change how we handle the HttpClient since the consensus is the create 1 instance and reuse it: http://stackoverflow.com/a/15708633/35389

### CODE REVIEW

**Comment by vasily-kirichenko** [comment]
I don't mind to remove the additional check. 
@dungpa what do you think?

**Comment by dungpa** [comment]
Sounds like a good idea.

File: src/FSharpVSPowerTools.Logic/FsiReferenceCommand.fs [hunk 1]
```diff
@@ -174,32 +174,25 @@ type FsiReferenceCommand(dte2: DTE2, mcs: OleMenuCommandService) =
...
-            let! project = Option.ofNull project
-            let! currentConfiguration = 
-                try Some (project.ConfigurationManager.ActiveConfiguration.ConfigurationName.ToLower())
-                with e -> 
-                    Logging.logError "[FsiReference] Cannot get current configuration name: %O" e
-                    None
-            let loadRefsFileName = sprintf "load-references-%s.fsx" currentConfiguration
+        Option.ofNull project
+        |> Option.iter (fun project ->
+            let loadRefsFileName = "load-references.fsx"
```
Comment (bleis-tift) [inline src/FSharpVSPowerTools.Logic/FsiReferenceCommand.fs]: @vasily-kirichenko  Are these changes really OK?
These changes look like to revert a3d828d170063ec6612951151dc53c7d2a794266 .

Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Logic/FsiReferenceCommand.fs]: Yes, it's. I'm just tired to commit the generated files each time I switch configuration and build.

Comment (TWith2Sugars) [inline src/FSharpVSPowerTools.Logic/FsiReferenceCommand.fs]: Whoops, didn't realise I commited this.

Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Logic/FsiReferenceCommand.fs]: Np, I cherry picked my commit and push it into master.

**Pull request merged**